### PR TITLE
Pull request 20181205

### PR DESCRIPTION
**Pull request by andrea-adami**
Build-fixes for kodi_18 for mips/mipsel and mips64/mips64el.

### CODE REVIEW

File: recipes-multimedia/kodi/kodi_18.bb [hunk 1]
```diff
@@ -55,6 +55,7 @@ DEPENDS += " \
             libssh \
             libtinyxml \
             libusb1 \
+            libxkbcommon \
```
Comment (koenkooi) [inline recipes-multimedia/kodi/kodi_18.bb:58]: This should go into 
```PACKAGECONFIG[x11] = "-DCORE_PLATFORM_NAME=x11,,libxinerama libxmu libxrandr libxtst glew"```

Comment (andrea-adami) [reply recipes-multimedia/kodi/kodi_18.bb:58]: right, do you prefer that I rebase or a further patch to fix it?

**Comment by koenkooi** [comment]
A rebase please.

**Comment by andrea-adami** [comment]
With:
PACKAGECONFIG[x11] = "-DCORE_PLATFORM_NAME=x11,,libxinerama libxkbcommon libxmu libxrandr libxtst glew"

Configure fails (qemumips is using gbm in my default build). 
Could NOT find Xkbcommon (missing: XKBCOMMON_LIBRARY XKBCOMMON_INCLUDE_DIR)
CMakeLists.txt:179 (core_require_dep)
Configuring incomplete, errors occurred!

Adding:
PACKAGECONFIG[gbm] = "-DCORE_PLATFORM_NAME=gbm -DGBM_RENDER_SYSTEM=gles,,libxkbcommon"

Then it builds

Searching around, it might be needed by wayland and gbm.
Finally, see this commit: https://github.com/xbmc/xbmc/commit/a7bb932a9281d14c4461e1afbd29b96af53e2588

Should we indeed add it unconditionally?
Cheers

**Comment by koenkooi** [comment]
Yes, adding it unconditionally would be best, I guess my build gets it through mesa

Verstuurd vanaf mijn iPad

**Pull request merged**